src/daft-micropartition/src/micropartition.rs

```diff
@@ -134,6 +134,28 @@ fn materialize_scan_task(
                 }) => {
                     let inference_options =
                         ParquetSchemaInferenceOptions::new(Some(*coerce_int96_timestamp_unit));
+
+                    let num_parallel_tasks = 8;
```

Comment:
Should this be pulled from somewhere rather than being hardcoded?

Reply:
Damn. I see it used to be hardcoded too lol. Perhaps we should at least add a TODO here that this is a magic number